### ISSUE TITLE
Phase "cancel" fired twice

### ISSUE THREAD

**Issue by BastienLeprince** [mattbryson/TouchSwipe-Jquery-Plugin#284]
When you add variables or console.logs in the "cancel" phase it fires twice.

**Comment by lawlite** [comment]
+1

For the moment, I skirted that with end-phase and distance options like this makeshift.

```javascript
// issue
$target.swipe({
    threshold: 100,
    swipeStatus: function(event, phase, direction, distance){
        if (phase == "cancel") {
            alert("Canceled"); // show twice
        }
    }
});

// makeshift
$target.swipe({
    threshold: 0,
    swipeStatus: function(event, phase, direction, distance){
        if (phase == "end") {
            if (distance < 100) { // instead of "threshold: 100"
                alert("Canceled"); // show once
            }
        }
    }
});
```

**Comment by mattbryson** [comment]
fixed in 1.6.15
